packages/label-sync/src/label-sync.ts

```diff
@@ -73,12 +77,14 @@ export = (app: Application) => {
     const { owner, repo } = context.repo();
     // TODO: Limit this to pushes that edit `labels.json`
     if (
-      owner === 'googleapis' &&
+      owner === 'sofisl' &&
       repo === 'repo-automation-bots' &&
       context.payload.ref === 'refs/heads/master'
     ) {
-      await refreshLabels();
+      await refreshLabels(context.github);
       // TODO: Use the GitHub installations API to retreive this list
+      //   (also, our QA indicates there's a good chance this request isn't
```

Comment:
I think we can get rid of thee comments, seems like your QA showed us things were working.